source/application/BioXAS/BioXASAppController.cpp

```diff
@@ -783,9 +814,11 @@ void BioXASAppController::setupGenericStepScanConfiguration(AMGenericStepScanCon
 
 		// Set scan detectors.
 
-		AMDetector *i0Detector = BioXASBeamline::bioXAS()->i0Detector();
-		if (i0Detector)
-			configuration->addDetector(i0Detector->toInfo());
+//		connect( BioXASBeamline::bioXAS()->defaultGenericScanDetectors(), SIGNAL(connected(bool)), this, SLOT(updateGenericScanConfigurationDetectors()) );
```

Comment:
I'd like to hang onto them, if that's okay. They are commented right now to avoid issues related to #2148 and I plan to use them when that issue is dealt with.